templates/download/shared/_server_weekly_news.html

```diff
@@ -0,0 +1,68 @@
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-12">
+      <h3>Get Ubuntu Server weekly news</h3>
```

Comment:
When I subscribe to Ubuntu Server weekly news, the new page invites me to subscribe to Ubuntu Server weekly news again, which is pointless. Either the form submission should not go to a new page, or this section should be hidden on the new page.

Reply:
Maybe in this instance, the "modal" thank you banner could replace the form (keeping the same dimensions) and contain a link to the PDF (with target="_blank", to not disrupt the OS download).